labshare-client/src/components/HelperForm.vue

```diff
@@ -1,89 +1,167 @@
+            type="password"
+            id="password"
+            v-model="passwordRepeat"
+            trim
+          ></b-form-input>
         </b-form-group>
       </template>
 
-      <h3>Kontakt Informationen</h3>
-      <b-form-group id="firstname" label="Vorname">
-        <b-form-input id="firstname" v-model="formData.contact.firstname" trim></b-form-input>
+      <h3>{{ $t("contactInfo") }}</h3>
+      <b-form-group id="firstname" :label="$t('firstName')">
+        <b-form-input
+          id="firstname"
+          v-model="formData.contact.firstname"
+          trim
+        ></b-form-input>
       </b-form-group>
 
-      <b-form-group id="lastname" label="Nachname">
-        <b-form-input id="lastname" v-model="formData.contact.lastname" trim></b-form-input>
+      <b-form-group id="lastname" :label="$t('lastName')">
+        <b-form-input
+          id="lastname"
+          v-model="formData.contact.lastname"
+          trim
+        ></b-form-input>
       </b-form-group>
 
-      <b-form-group id="phone" label="Telefonnummer">
-        <b-form-input id="phone" v-model="formData.contact.phone" trim></b-form-input>
+      <b-form-group id="phone" :label="$t('phone')">
+        <b-form-input
+          id="phone"
+          v-model="formData.contact.phone"
+          trim
+        ></b-form-input>
       </b-form-group>
 
-      <h3>Wohnort</h3>
-      <b-form-group id="city" label="Ort">
-        <b-form-input id="city" v-model="formData.address.city" trim></b-form-input>
+      <h3>{{ $t("address") }}</h3>
+      <b-form-group id="city" :label="$t('city')">
+        <b-form-input
+          id="city"
+          v-model="formData.address.city"
+          trim
+        ></b-form-input>
       </b-form-group>
 
-      <b-form-group id="zipcode" label="Postleitzahl">
-        <b-form-input id="zipcode" v-model="formData.address.zipcode" trim></b-form-input>
+      <b-form-group id="zipcode" :label="$t('postCode')">
+        <b-form-input
+          id="zipcode"
+          v-model="formData.address.zipcode"
+          trim
+        ></b-form-input>
       </b-form-group>
 
-      <h3>Fähigkeiten</h3> 
-      <b-form-group id="organization" label="Organisation">
-        <b-form-input id="organization" v-model="formData.organization" trim></b-form-input>
+      <h3>{{ $t("competencies") }}</h3>
+      <b-form-group id="organization" :label="$t('organization')">
+        <b-form-input
+          id="organization"
+          v-model="formData.organization"
+          trim
+        ></b-form-input>
       </b-form-group>
 
-      <b-form-group label="Erfahrungen in folgenden Bereichen">
+      <b-form-group :label="$t('skills')">
         <b-form-checkbox-group
           id="skills"
           v-model="formData.details.skills"
-          :options="labSkills"
           name="name"
-          stacked></b-form-checkbox-group>
+          stacked
+        >
+          <b-form-checkbox
+            v-for="(skill, i) in labSkills"
```

Comment:
Now that I look at it again, `labSkills` could probably just be a string array, but it's not worth refactoring for its own sake (and arguably the object still has expressive value)